pyexcelerate/Worksheet.py

```diff
@@ -198,7 +200,10 @@ def __get_cell_data(self, cell, x, y, style):
             z = '"><f>%s</f></c>' % (cell[1:]) # Remove equals sign.
         elif type == DataTypes.BOOLEAN:
             z = '" t="b"><v>%d</v></c>' % (cell)
-
+        
+        if z is None:
```

Comment:
Can this just be an `else` block with the above conditional and avoid having to declare `z = None`?